akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/SecurityDirectivesSpec.scala

```diff
@@ -18,8 +18,8 @@ class SecurityDirectivesSpec extends RoutingSpec {
   val doOAuth2Auth = authenticateOAuth2PF("MyRealm", { case Credentials.Provided(identifier) ⇒ identifier })
   val authWithAnonymous = doBasicAuth.withAnonymousUser("We are Legion")
 
-  val basicChallenge = HttpChallenges.basic("MyRealm")
-  val oAuth2Challenge = HttpChallenges.oAuth2("MyRealm")
+  val basicChallenge = HttpChallenge("Basic", "MyRealm")
+  val oAuth2Challenge = HttpChallenge("Bearer", "MyRealm")
 
```

Comment:
Rolling back the possibly superfluous factory methods for Basic and OAuth2 challenges